force-app/main/default/lwc/primaryAffiliationsModalBody/primaryAffiliationsModalBody.html

```diff
@@ -0,0 +1,37 @@
+<template>
+    <div class="slds-grid slds-grid--vertical slds-grid_vertical-align-start slds-wrap">
```

Comment:
This section likely needs a little padding so the combobox dropdowns resolve and the text is not quite so squished together. Look at the mock for sizing targets and the lightning design system modal documentation for how to get modal sizing. We may want a medium or large in this case.